src/features/targeting/hocs/withTargetedClickThrough.js

```diff
@@ -22,6 +22,15 @@ function withTargetedClickThrough<Config>(
     |}) => {
         const { canShow, onComplete, onClose, onShow } = useTargetingApi();
 
+        const handleClose = () => {
+            onClose();
+            // $FlowFixMe Flow does not yet support method or property calls in optional chains.
+            if (rest?.onDismiss) {
```

Comment:
is there a reason why onDismiss should not be defined in the TargetedComponentProps ?